integration/tests/main/01-missing-permissions.spec.js

```diff
@@ -28,6 +28,8 @@ test.describe("Limited user simple deployments", () => {
     await releaseNameLocator.fill(utils.getRandomName("test-01-release"));
     await page.locator('cds-button:has-text("Deploy")').click();
 
+    // Assertions
+    await page.waitForSelector(".alert-items .alert-text");
```

Comment:
So it was just a kind of race condition, that's why retrying made it to work?

Reply:
Yeah, I would say that if the error message was appearing almost immediately, the problem was hidden. It makes total sense to have this `waitForSelector`, as neither `page.locator` nor `expect(errorLocator)` wait for the error to be there.